sql/migrations/20210901122902_world.sql

```diff
@@ -0,0 +1,39 @@
+
+-- Position of Gazban on spawn
+DELETE FROM `event_scripts` WHERE `id`=364;
+INSERT INTO `event_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES (364, 5, 10, 2624, 900000, 0, 0, 0, 0, 0, 0, 0, 0, -1, 1, -12167.9, 630.3, -63.78, 3.65, 0, 'Summon Gazban From Altar of the Tides');
+
+-- Change spell cooldown on Catelyns Blade, so you cant spam spawn Gazban
+UPDATE item_template SET class=12, subclass=0, name='Catelyn''s Blade', description='', display_id=7069, quality=1, flags=2112, buy_count=1, buy_price=0, sell_price=0, inventory_type=0, allowable_class=-1, allowable_race=-1, item_level=1, required_level=0, required_skill=0, required_skill_rank=0, required_spell=0, required_honor_rank=0, required_city_rank=0, required_reputation_faction=0, required_reputation_rank=0, max_count=1, stackable=1, container_slots=0, stat_type1=0, stat_value1=0, stat_type2=0, stat_value2=0, stat_type3=0, stat_value3=0, stat_type4=0, stat_value4=0, stat_type5=0, stat_value5=0, stat_type6=0, stat_value6=0, stat_type7=0, stat_value7=0, stat_type8=0, stat_value8=0, stat_type9=0, stat_value9=0, stat_type10=0, stat_value10=0, delay=0, range_mod=0.0, ammo_type=0, dmg_min1=0.0, dmg_max1=0.0, dmg_type1=0, dmg_min2=0.0, dmg_max2=0.0, dmg_type2=0, dmg_min3=0.0, dmg_max3=0.0, dmg_type3=0, dmg_min4=0.0, dmg_max4=0.0, dmg_type4=0, dmg_min5=0.0, dmg_max5=0.0, dmg_type5=0, block=0, armor=0, holy_res=0, fire_res=0, nature_res=0, frost_res=0, shadow_res=0, arcane_res=0, spellid_1=3678, spelltrigger_1=0, spellcharges_1=0, spellppmrate_1=0.0, spellcooldown_1=30000, spellcategory_1=0, spellcategorycooldown_1=-1, spellid_2=0, spelltrigger_2=0, spellcharges_2=0, spellppmrate_2=0.0, spellcooldown_2=-1, spellcategory_2=0, spellcategorycooldown_2=-1, spellid_3=0, spelltrigger_3=0, spellcharges_3=0, spellppmrate_3=0.0, spellcooldown_3=-1, spellcategory_3=0, spellcategorycooldown_3=-1, spellid_4=0, spelltrigger_4=0, spellcharges_4=0, spellppmrate_4=0.0, spellcooldown_4=-1, spellcategory_4=0, spellcategorycooldown_4=-1, spellid_5=0, spelltrigger_5=0, spellcharges_5=0, spellppmrate_5=0.0, spellcooldown_5=0, spellcategory_5=0, spellcategorycooldown_5=0, bonding=4, page_text=0, page_language=0, page_material=0, start_quest=0, lock_id=0, material=0, sheath=0, random_property=0, set_id=0, max_durability=0, area_bound=0, map_bound=0, duration=0, bag_family=0, disenchant_id=0, food_type=0, min_money_loot=0, max_money_loot=0, extra_flags=0, other_team_entry=1 WHERE entry=4027 AND patch=0;

```

Comment:
Editing item template is not blizzlike. Instead you should add a condition to the script that summons him to only summon him if he is not already there.

Reply:
Thanks for the feedback, New commit has the condition applied.

Reply:
UPDATE creature_template SET faction=35, unit_flags=32768, auras='6523', movement_type=2 WHERE entry=2624;

This guy should be on an island too:

>     He yelled "What is happening? What force draws me to the Altar of Tides!" right after he gained Phase Out (according to SpellAlert).


He also has waypoints but the fight or "summon" is not sniffed..
I Think it would be good to sniff the quest and hit him without any weapon to see on how much % health he transforms.

Reply:
@Oroxzy Fix your account and sniff it on the PTR :)

Reply:
I will😬